lib/chromedriver.js

```diff
@@ -514,11 +514,7 @@ class Chromedriver extends events.EventEmitter {
     let cmd;
```

Comment:
I'd rather use the ternary operator here

Reply:
Something like now?